### ISSUE TITLE
When upgrading from 0.4 to 0.5, get ImportError

### ISSUE THREAD

**Issue by rtibbles** [learningequality/kolibri#1859]
## Summary

Upgrading from 0.4 to 0.5 produces an import error due to the disappearance of two plugins.

## System information

 - Version: v0.5.0-beta4
 - Operating system: Windows 10
 - Browser: N/A

## Traceback or relevant snippet from server.log or browser console

```
C:\Python27\Scripts>kolibri start
Traceback (most recent call last):
  File "c:\python27\lib\runpy.py", line 174, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "c:\python27\lib\runpy.py", line 72, in _run_code
    exec code in run_globals
  File "C:\Python27\Scripts\kolibri.exe\__main__.py", line 9, in <module>
  File "c:\python27\lib\site-packages\kolibri\utils\cli.py", line 532, in main
    initialize(debug=debug)
  File "c:\python27\lib\site-packages\kolibri\utils\cli.py", line 134, in initialize
    django.setup()
  File "c:\python27\lib\site-packages\kolibri\dist\django\__init__.py", line 18, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "c:\python27\lib\site-packages\kolibri\dist\django\apps\registry.py", line 85, in populate
    app_config = AppConfig.create(entry)
  File "c:\python27\lib\site-packages\kolibri\dist\django\apps\config.py", line 123, in create
    import_module(entry)
  File "c:\python27\lib\importlib\__init__.py", line 37, in import_module
    __import__(name)
ImportError: No module named video_mp4_render
```

## How to reproduce

1. Install v0.4.x
2. Upgrade to v0.5.0-beta4
3. Try to run the server

## Real-life consequences (anything community should be aware of, for instance how it affects your deployment)

Upgrading becomes significantly more painful. Should rather show a warning about non-existent plugins and remove them from the configuration.

Originally reported by @laurenlichtman

**Comment by benjaoming** [comment]
Yeah, that seems like the most expected behaviour, but it would be nice if we could also get the warnings to display in the browser.. do we use `django.contrib.messages` by any chance?

**Comment by rtibbles** [comment]
I don't think we are currently doing so. Not sure how useful this warning is to the end user, as opposed to the person upgrading, who presumably would see the terminal or log?

**Comment by benjaoming** [comment]
> opposed to the person upgrading, who presumably would see the terminal or log

Nah, if the install view .deb or .exe and have a script that just starts KA Lite on boot, they won't see these messages.

**Comment by benjaoming** [comment]
Currently, `conf` sets `config['INSTALLED_APPS']` - and this is appended in `settings.INSTALLED_APPS`.

This is a bit sub-optimal, because while we might wanna switch off plugins from our core that have been removed, we should also fail when other critical aspects are unavailable.

I'll just add a comment in the `conf` module about this for now, since we are going to do a real overhaul of the way that settings work anyways.

**Comment by rtibbles** [comment]
Fixed in #1902.
